src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/JsonEncodingStreamWrapper.cs

```diff
@@ -103,7 +103,6 @@ public override long Position
         {
             get
             {
-#pragma warning suppress 56503 // The contract for non seekable stream is to throw exception
                 throw new NotSupportedException();
```

Comment:
Nit: the formatting can now be changed to match the rest of the file, e.g. `get { throw new NotSupportedException(); }`